scripts/deploy.sh

```
@@ -6,77 +6,104 @@ cd "$APP_DIR"
+deploy_process() {
+  local PORT=$1
+  local USE_OTEL=$2
+  local LOG_FILE=$APP_DIR/app-$PORT.log
+
+  echo "=== [$PORT] 배포 시작 ==="
+
+  echo "> [$PORT] 실행 중인 프로세스 종료"
+  OLD_PID=$(pgrep -f "server.port=$PORT" || true)
+  if [ -n "$OLD_PID" ]; then
+    echo "> [$PORT] 종료: $OLD_PID"
+    kill -15 $OLD_PID
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)